### DISCUSSION TITLE
Mamba and Conda... again!

### DISCUSSION THREAD

**Discussion by peanutfun** [CLIMADA-project/climada_python#853] [General]
Conda has actually [switched to the libmamba solver](https://docs.conda.io/projects/conda/en/stable/release-notes.html#with-this-23-10-0-release-we-are-changing-the-default-solver-of-conda-to-conda-libmamba-solver) as of October 2023. This means that there should be practically no difference in environment solving and the associated timings between Conda and Mamba. The inconsistencies and solver errors we often faced might actually have been the result of using both the `defaults` and the `conda-forge` channels simultaneously, although they are considered incompatible. This was reported in #843 and is now thankfully fixed.

Currently everything works fine, but I am starting this discussion to keep track of the development over at Mamba. In particular, there is a discussion if Mamba should be deprecated in favor of Conda and Micromamba, see https://github.com/mamba-org/mamba/pull/3054. To summarize the discussion as of 2024-02-21:
* Mamba and Conda now use the same solver but show some inconsistencies in CLI usage. Having only one option might reduce confusion among users.
* The proper lightweight alternative to Conda is Micromamba, not Mamba.

**Comment by peanutfun** [comment]
How to switch back to Conda:
* Replace all `mamba` commands with `conda` in the installation instructions
* Tell users to install Conda. They can choose between [Miniconda](https://docs.anaconda.com/free/miniconda/) and [Anaconda](https://www.anaconda.com/download), see https://docs.conda.io/projects/conda/en/stable/user-guide/install/index.html *(edit: Miniforge for conda/mamba is also fine)*
* Add `conda-forge` as highest-priority channel following these instructions: https://conda-forge.org/docs/user/introduction/#how-can-i-install-packages-from-conda-forge

@chahank

**Reply by chahank** [reply]
Thanks, I can prepare a PR to update the installation.

**Reply by chahank** [reply]
Starting working on the branch [feature/installation_back_to_conda](https://github.com/CLIMADA-project/climada_python/tree/feature/installation_back_to_conda). It is in progress. Any inputs are welcome.

**Reply by bguillod** [reply]
What is the status there - this has not yet been merged and is not part of the most recent release (5.0.0), correct?

**Reply by peanutfun** [reply]
@bguillod Correct. We are currently facing some issues with Conda again, which might postpone us switching back. See #913
